lib/policy_machine_storage_adapters/active_record.rb

```diff
@@ -1077,6 +1085,25 @@ def build_accessible_object_scope(associations, options = {})
       end
     end
 
+    def use_accessible_objects_function?(options)
+      PolicyMachineStorageAdapter.postgres? &&
+        options[:direct_only] &&
+        options[:ignore_prohibitions] &&
```

Comment:
actually @kjerinsky-mdsol i know i just approved but what do you think
```
        options[:direct_only] == true &&
        options[:ignore_prohibitions] == true &&
```

Reply:
d5f0d2a